Libraries/LibRequests/Request.cpp

```diff
@@ -9,6 +9,21 @@
 
 namespace Requests {
 
+#if defined(AK_OS_WINDOWS)
+ErrorOr<NonnullOwnPtr<ReadStream>> ReadStream::try_create(int const reader_fd)
```

Comment:
style nit: Let's put the Windows guards inside the function body, so we don't need to repeat the function declarations.

style nit 2: Making the `reader_fd` parameter const just adds noise, IMO.